addons/rating/models/mail_message.py

```diff
@@ -45,3 +45,5 @@ def _to_store(self, store: Store, /, *, fields=None, **kwargs):
             for records in self._records_by_model_name().values():
                 if issubclass(self.pool[records._name], self.pool["rating.mixin"]):
                     store.add(records, fields=["rating_avg", "rating_count"], as_thread=True)
+                    if records._allow_publish_rating_stats():
```

Comment:
This should actually be a widget option... this is not really model oriented. Especially allow implies something based on rights, while it is just a choice in portal display as far as I understand the feature. There should not have any model code if possible (except store management checking if somehow option was chosen).

Reply:
`display_rating` is still a widget option which is used to display rating on the composer in portal, although we wanted to send the data related to rating summary via store if the specific model uses this part of the widget, not for all. If we don't use model code, we have to send the `request for the option` through the routes and then put it in context so that it is accessible when creating the store if it's better?